containers/package-builder/clone-or-update.sh.j2

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+repository=$1
+directory=$2
+
+# Check if a directory exists
+if [ ! -d "$directory" ] && [[ "$repository" == "/var/concent-data/concent" ]]; then
+    git clone "$repository" "$directory"
+    cd "$directory"
+    git checkout {{ concent_version }}
```

Comment:
The code in both `if`s is now identical. You can replace it with a single `if`.

Reply:
It's not possible to do that. How would you recognize which template variable use ?